usaon_vta_survey/templates/macros/user_login_management_buttons.j2

```diff
@@ -1,7 +1,13 @@
 {% macro user_login_buttons(current_user) -%}
   {% if current_user.is_authenticated %}
-    <a href="{{url_for('profile', user_id=current_user.id)}}">Profile</a>
-    <a href="{{url_for('logout')}}">Logout</a>
+    {% if current_user.role=='admin' %}
+      <a href="{{url_for('profile', user_id=current_user.id)}}">Profile</a>
+      <a href="{{url_for('view_users')}}">Users</a>
```

Comment:
Might need to consider renaming this file again haha

Reply:
Hmm... `login_button_or_private_nav_buttons`?